roles/openshift_noobaa/defaults/main.yml

```
@@ -0,0 +1,4 @@
+---
+# defaults file for openshift_noobaa
+IMAGE: noobaaimages.azurecr.io/noobaa/nbserver:latest
```

```suggestion
noobaa_image: noobaaimages.azurecr.io/noobaa/nbserver:latest
```
role-level vars should be lowercase and prefixed with role name, preferably